src/browser/documentation/sidebar-guides/concepts.tsx

```diff
@@ -22,144 +22,167 @@ import React from 'react'
 import ManualLink from 'browser-components/ManualLink'
 import { BuiltInGuideSidebarSlide } from '../../modules/Carousel/Slide'
 
-const title = 'Concepts'
+const title = 'Concepts Guide'
 const category = 'guides'
 const slides = [
   <BuiltInGuideSidebarSlide key="first">
-    <h3>Graph Fundamentals</h3>
-    <p className="lead">Basic concepts to get you going.</p>
+    <h3>Property graph model concepts</h3>
+    <p className="lead">
+      <em>Basic concepts to get you going</em>
```

Comment:
Basic concepts to get you going.

There is a mixture of either no `.` or a `.`

I think it is best to always use `.`

Reply:
Yeah, we have to decide on whether to have dots or not. I prefer to see them as subtitles instead of standard sentences.